### PR TITLE
create a personal profile

### PR DESCRIPTION
**Pull request by officialrajdeepsingh**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5335"></a>

### CODE REVIEW

File: data/officialrajdeepsingh.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      "url": "https://github.com/officialrajdeepsingh",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "LinkedIn: Let's connect",
+      "url": "https://www.linkedin.com/in/officalrajdeepsingh/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Freecode camp",
```
Comment (shubhsharma19) [inline data/officialrajdeepsingh.json:30]: You might want to remove that extra space?

**Review by shubhsharma19 — APPROVED** [review]
LGTM 🎉

**Review by SurajPratap10 — DISMISSED** [review]
LGTM

**Review by Arindam200 — DISMISSED** [review]

**Review by himanshu1221 — APPROVED** [review]

File: data/officialrajdeepsingh.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    },
+    {
+      "name": "Freecodecamp",
+      "url": "https://www.freecodecamp.org/news/author/officialrajdeepsingh/",
+      "icon": "FaFreeCodeCamp"
+    },
+    {
+      "name": "website: Let's connect",
+      "url": "https://www.linkedin.com/in/officalrajdeepsingh/",
+      "icon": "FaGlobe"
```
Comment (kumarsonsoff3) [inline data/officialrajdeepsingh.json:37]: Looks good, but it seems like you've added the same link twice. You might want to add your website link here.

Comment (officialrajdeepsingh) [reply data/officialrajdeepsingh.json]: Now fix the issue

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**